### DISCUSSION TITLE
Domain qwernik.md gets -1 in domain_expiry_days

### DISCUSSION THREAD

**Discussion by ryazantseff** [caarlos0/domain_exporter#321] [Q&A]
Please help, seems like parsing issue.
Here is whois response:
```
Domain   name:  qwernik.md
Domain  state:  OK, Delegated
Registrar:      HOST.MD 

Registered on:  2021-12-16
Expires    on:  2024-12-16

Nameserver:     ns1.host.md
Nameserver:     ns2.host.md
```

**Comment by vtmocanu** [comment]
are you trying from the same ip as the exporter?
i had the same issue, but it think it's a rate limit from the whois server, because from the cli stopped working as well.
